bin/millau/runtime/src/lib.rs

```diff
@@ -225,7 +225,7 @@ impl pallet_bridge_call_dispatch::Trait for Runtime {
 	type Event = Event;
 	type MessageId = (bp_message_lane::LaneId, bp_message_lane::MessageNonce);
 	type Call = Call;
-	type SourceChainAccountPublic = MultiSigner;
+	type SourceChainAccountId = MultiSigner;
```

Comment:
We should use `<T as frame_system::Trait>::AccountId` instead of this type, also it's incorrect to use `MultiSigner` here, since it's public keys.

Reply:
`MultiSigner` implements [`IdentifyAccount`](https://crates.parity.io/sp_runtime/enum.MultiSigner.html#impl-IdentifyAccount), so why isn't it fine to use it here?

Reply:
Because we don't want `IdentifyAccount` here :) We want `IdentifyAccount::AccountId` here, since it's a `SourceChainAccountId`, not public key.